### ISSUE TITLE
[WSL] VS Code not in sync with project actual state.

### ISSUE THREAD

**Issue by edassis** [microsoft/vscode#110638]
Issue Type: <b>Bug</b>

When using WSL, sometimes the UI not show the actual state of the files. I need to refresh the explorer to show the actual state of my project. Besides, seens that vs code not know about changes in the files inside .vscode/, when I create a new launch.json per example, the run view not show me the option to execute the new file, I have to reopen vs code.

VS Code version: Code 1.51.1 (e5a624b788d92b8d34d1392e4c4d9789406efe8f, 2020-11-10T23:34:32.027Z)
OS version: Windows_NT x64 10.0.19041
Remote OS version: Linux x64 4.19.84-microsoft-standard

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 2600X Six-Core Processor            (12 x 3600)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.95GB (8.19GB free)|
|Process Argv|C:\Users\eduar\OneDrive\Documentos\Programacao\Competitive --crash-reporter-id 2cc80329-f9c1-48b1-ab0f-8c74ae104120|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 4.19.84-microsoft-standard|
|CPUs|AMD Ryzen 5 2600X Six-Core Processor (12 x 3600)|
|Memory (System)|12.45GB (11.37GB free)|
|VM|0%|
</details><details><summary>Extensions (15)</summary>

Extension|Author (truncated)|Version
---|---|---
language-x86-64-assembly|13x|2.3.0
emojisense|bie|0.8.0
vsc-material-theme|Equ|33.0.0
vsc-material-theme-icons|equ|1.2.0
mips|kda|0.0.5
remote-wsl|ms-|0.51.3
vscode-icons|vsc|11.0.0
material-theme|zhu|3.9.3
TabOut|alb|0.2.0
doxdocgen|csc|1.1.0
gitlens|eam|10.2.3
code-runner|for|0.11.1
tic80-vscode|Gi9|1.7.0
cpptools|ms-|1.1.1
markdown-all-in-one|yzh|3.4.0

(4 theme extensions excluded)

</details>

**Comment by Stanzilla** [comment]
Yeah I noticed this as well, any file changes done outside of VSC are not recognized without a refresh. As an example if you create a new file. If you open the command palette and enter the file name it shows up and you can open it but the explorer list won't refresh.
